lib/Doctrine/Migrations/Metadata/Storage/MetadataStorageConfigration.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Metadata\Storage;
+
+interface MetadataStorageConfigration
```

Comment:
Apart from the typo (thanks @lcobucci), are there any methods we can/should add to the interface?

Copied from https://github.com/lcobucci/doctrine-migrations/pull/1/commits/1f59f350aae1d96ba10e7c3e645fa34994787fcc

Reply:
will fix

Reply:
Since each "metadata storage" might have complete different ways to store data, and the configurations for that storage are used only buy the storage, there are no shared configs that could be placed in this interface


This should answer to https://github.com/doctrine/migrations/pull/858#discussion_r345026244